.github/workflows/build-deb.yaml

```diff
@@ -0,0 +1,112 @@
+name: Build debian packages
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
```

Comment:
I'm not a big fan of having this running on every single PR and push to main, tbh. I think it's better to trigger this only if there were any changes on `debian/`, otherwise it will increase our CI time to run even more for little gain.

Reply:
Not sure if there's a way in github to run only a workflow when a PR is approved? As that would be a way to prevent problems to hit to main without having to test all the PR's from the very first moment.

Reply:
You can add a job that relies on the completion of another one by setting the `needs:` field, but I still think that this is not something we need to constantly watch for.

Reply:
Also, I'm afraid this kind of action might give users the wrong impression that the code in `main` is always "ready for usage" when, in reality, this kind of project relies a lot on manual tests to ensure none of our dependencies regressed.

Reply:
No, well that's not the point IMHO but at the same it's useful for testing having packages ready all the time.

But well, if someone think that, then is wrong as it would be wrong in any other project doing nightly builds.

However, I can see if can change this to reduce the scope of its action. I guess we'll do this after FF though since this still depends on https://github.com/canonical/desktop-engineering/pull/28

Reply:
This is a CI related PR, so it can definitely wait and be done after FF.

Reply:
Yeah, the reason for that was to get this early to be confident on uploading things working stuff to the distro, but we can still append these commits to temporary branches to check things for now.

Reply:
This is now handled, as per https://github.com/ubuntu/authd/pull/130/commits/dee7b3a39e59178d3516bd36ac9fb81c254784d1